crates/ruff/src/rules/isort/settings.rs

```diff
@@ -134,6 +134,15 @@ pub struct Options {
             force-to-top = ["src"]
         "#
     )]
+    /// Sort imports taking into account case sensitivity.
+    pub case_sensitive: Option<bool>,
+    #[option(
+        default = r#"false"#,
+        value_type = "bool",
+        example = r#"
+            case-sensitive = true
+        "#
+    )]
```

Comment:
This is slightly out-of-order, I think this needs to be _above_ the docstring ("/// Sort imports taking into account case sensitivity.").